docker-compose.yml

```diff
@@ -3,8 +3,12 @@
 
 web:
   image: samos123/drupal:7.x
+  #build: .
```

Comment:
please remove any commented testing-options here. you may add this as an example in the `README.md`, but as it's equivalently configured like mysql, a linkt to the docker library's postgres-image should suffice.